queries.sql

```diff
@@ -1,17 +1,62 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * FROM animals WHERE name LIKE '%mon';
+BEGIN TRANSACTION;
 
-SELECT name FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+  UPDATE animals SET species='unspecified';
 
-SELECT name FROM animals WHERE neutered=true AND escape_attempts<3;
+  SELECT * FROM animals ORDER BY id; 
 
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+  SELECT species from animals;
+  
+ROLLBACK;
 
-SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5;
 
-SELECT * FROM animals WHERE neutered=true;
+BEGIN TRANSACTION;
 
-SELECT * FROM animals WHERE name!='Gabumon';
+  UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
 
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg<=17.3;
+  UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+  SELECT species from animals;
+
+COMMIT TRANSACTION;
+
+
+BEGIN TRANSACTION;
+
+  DELETE FROM animals;
+
+  SELECT COUNT(*) FROM ANIMALS;
+
+ROLLBACK;
+
+Begin TRANSACTION;
+
+  DELETE FROM animals WHERE date_of_birth  >  '2022-01-01';
+
+  SAVEPOINT date_filter;
+
+  UPDATE animals SET weight_kg = (weight_kg * -1);
+
+  ROLLBACK TO date_filter;
+
+  UPDATE animals SET weight_kg = (weight_kg * -1) WHERE weight_kg < 0;
+  
+COMMIT TRANSACTION;
+
```

Comment:
- [ ] You should add screenshots of your transaction results to the pull request description. :+1: